pg_mooncake/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+
+sudo apt-get update
+sudo apt-get install -y docker.io
+sudo apt-get install -y postgresql-client
+
+wget --no-verbose --continue https://datasets.clickhouse.com/hits_compatible/athena/hits.parquet
+sudo docker run -d --name pg_mooncake -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust - -v ./hits.parquet:/tmp/hits.parquet mooncakelabs/pg_mooncake
+
+sleep 5
+psql postgres://postgres:pg_mooncake@localhost:5432/postgres -f create.sql
```

Comment:
This returned

```
65b913e46da5fd63729ffcd82a3d8cc92c50012077a4b9180081ef5e1d7414a2
psql:pg_mooncake/create.sql:108: ERROR:  access method "columnstore" does not exist
psql:pg_mooncake/create.sql:112: ERROR:  relation "hits" does not exist
LINE 1: insert into hits select WatchID, JavaEnable, Title, GoodEven...
```

At that point, I stopped checking further. Could you please fix these issues?